packages/combobox/src/Combobox/Combobox.spec.tsx

```diff
@@ -153,6 +153,33 @@ describe('packages/combobox', () => {
         );
       });
 
+      test('option renders description', () => {
+        const { openMenu } = renderCombobox(select);
+        const { optionElements } = openMenu();
+        expect(optionElements?.[0]).toHaveTextContent(
+          defaultOptions[0].description as string,
+        );
+      });
+
+      test('option fires onClick callback', () => {
+        const onClick = jest.fn();
+
+        const options: Array<OptionObject> = [
+          {
+            value: 'paragraph',
+            displayName: 'display name',
+            isDisabled: false,
+            onClick,
+          },
+        ];
+
+        const { openMenu } = renderCombobox(select, { options });
+        const { optionElements } = openMenu();
+        const [optionEl] = Array.from(optionElements!);
+        userEvent.click(optionEl);
+        expect(onClick).toHaveBeenCalledTimes(1);
```

Comment:
can we check that the callback is called with the arguments that we expect?

Reply:
and confirm that this does not conflict with/break the existing event callbacks being passed to options and their parent